sdk/utils.ts

```diff
@@ -142,4 +144,12 @@ export const setPrivateFindings = (isPrivate: boolean) => {
 
 export const isPrivateFindings = () => {
   return IS_PRIVATE_FINDINGS
+}
+
+export const getFortaAlerts = async (query: AlertQueryOptions): Promise<AlertsResponse> => {
```

Comment:
i would call this `getAlerts` or `queryAlerts` (probably dont need the Forta name in there)